content/applications/services/helpdesk/overview/sla.rst

```diff
@@ -1,47 +1,194 @@
    :align: center
-   :height: 330
-   :alt: View of a ticket’s form emphasizing a satisfied SLA in Odoo Helpdesk
+   :alt: A Helpdesk ticket with two SLA tags attached.
+
+   This Helpdesk ticket has two SLA policies applied to it. One of the policies has been satisfied,
+   so the tag has turned green. The other policy is in progress.
 
-SLA Analysis
-============
+.. important::
+   If a ticket fits the criteria for more than one :abbr:`SLA (Service Level Agreement)`, the
+   earliest occurring deadline is displayed on the ticket. After that deadline has passed, the next
+   deadline is displayed.
 
-Go to :menuselection:`Helpdesk --> Reporting --> SLA Status Analysis`. Apply *Filters* and
-*Group by* to identify tickets that should be prioritized and keep track of upcoming deadlines.
+   .. image:: sla/sla-deadline.png
+      :align: center
+      :alt: A Helpdesk ticket with emphasis on the deadline field.
 
-.. image:: sla/sla_analysis.png
+If the :abbr:`SLA (Service Level Agreement)` deadline passes and the ticket has not moved to the
+:guilabel:`Reach Stage`, the :abbr:`SLA (Service Level Agreement)` tag turns red. After the
+:abbr:`SLA (Service Level Agreement)` has failed, the red tag stays on the ticket, even after the
+ticket is moved to the :guilabel:`Reach Stage`.
+
+Analyze SLA performance
+=======================
+
+The :guilabel:`SLA Status Analysis` report tracks how quickly an :abbr:`SLA (Service Level
+Agreement)` is fulfilled, as well as the performance of individual team members. Navigate to the
+report, and corresponding pivot table, by going to :menuselection:`Helpdesk app --> Reporting -->
+SLA Status Analysis`.
+
+Pivot view
+----------
+
+By default, the report displays in a :guilabel:`Pivot` view. Any :abbr:`SLA (Service Level
+Agreement)` policies in the database with tickets that failed to fulfill a policy, are in progress,
+or have satisfied a policy are listed.
+
+.. image:: sla/sla-status-analysis.png
    :align: center
-   :alt: View of the SLA status analysis page emphasizing the group by option in Odoo Helpdesk
+   :alt: The pivot view of the SLA Analysis report.
+
+To change the display, or add additional measurements, click the :guilabel:`Measures` button to
+reveal a drop-down menu of reporting criteria, and choose from the options available.
+
+Whenever a measurement is picked, a :guilabel:`✔️ (checkmark)` icon appears in the drop-down menu to
+indicate that the measurement is included, and a corresponding new column emerges in the pivot table
+to show the relevant calculations.
+
+To add a group to a row or column, click the :guilabel:`➕ (plus)` icon next to the policy name and
+then select one of the groups. To remove one, click the :guilabel:`➖ (minus)` icon next to the
+policy name.
+
+Graph view
+----------
+
+The :guilabel:`SLA Status Analysis` report can also be viewed as a :guilabel:`Bar Chart`,
+:guilabel:`Line Chart`, or :guilabel:`Pie Chart`. Toggle between these views by first selecting the
```

Comment:
```suggestion
The :guilabel:`SLA Status Analysis` report can also be viewed as a *Bar Chart*,
*Line Chart*, or *Pie Chart*. Toggle between these views by first selecting the
```
Again, suggesting the use of ** over GUI because, until they click the graph button (which *does* appear on the screen at this point), those options do not appear on the screen.